flight/Modules/Actuator/actuator.c

```diff
@@ -65,43 +64,57 @@
 
 #define TASK_PRIORITY PIOS_THREAD_PRIO_HIGHEST
 #define FAILSAFE_TIMEOUT_MS 100
+
+#ifndef MAX_MIX_ACTUATORS
```

Comment:
Allows specifying a per-target number of actuators, to lower memory use and computation requirements.  Will help Lux (4), Seppuku (8), etc.   Currently not employed.